celery/backends/redis.py

```diff
@@ -453,7 +458,7 @@ def apply_chord(self, header_result_args, body, **kwargs):
     def _chord_zset(self):
         return self._transport_options.get('result_chord_ordered', True)
 
-    @cached_property
+    @property
```

Comment:
are you sure about this change?

Reply:
When I have this cached_property 4 test are doing to failure:
```python
======================================================================= short test summary info ========================================================================
FAILED t/unit/backends/test_redis.py::test_RedisBackend_chords_simple::test_on_chord_part_return__unordered - AssertionError: assert 0
FAILED t/unit/backends/test_redis.py::test_RedisBackend_chords_simple::test_on_chord_part_return_no_expiry__unordered - AssertionError: assert 0
FAILED t/unit/backends/test_redis.py::test_RedisBackend_chords_simple::test_on_chord_part_return__ChordError__unordered - TypeError: '_ContextMock' object is not subscriptable
FAILED t/unit/backends/test_redis.py::test_RedisBackend_chords_simple::test_on_chord_part_return__other_error__unordered - TypeError: '_ContextMock' object is not subscriptable
=============================================================== 4 failed, 82 passed in 63.34s (0:01:03) ================================================================
```
Its looks like **_transport_options** is setup once and every test use it (doesn't update their custom setup).
Also I don't see reason that should be cached if:
1. It is in settings object so all worker have their own copy.
2. Get something form dict don't cost much for computation.

Maybe something I have missed or I haven't know and any advice how to fix will be welcome and I change if back :)